src/common/Types.ts

```diff
@@ -1141,7 +1145,7 @@ export enum LinkingErrorCode {
 
 export interface LinkingErrorInfo {
     code: LinkingErrorCode;
-    url: string;
+    url: string|undefined;
```

Comment:
We recently changed links so their URL was required, so this should just be "string".